style/style.css

```diff
@@ -116,190 +132,118 @@
-}
-.interface-app__explore{
-  padding-top: 150px;
-  padding-bottom: 25px;
-  width: 700;
-  color: #FA4A0C;
+.description-section__columns.reverse .description-section__column-img {
+  justify-content: flex-end;
+}
+.description-section__benefit-text {
+  font-weight: 700;
+  color: #fa4a0c;
   font-size: 24px;
 }
-.interface-app__advantages{
-  padding-top: 20px;
-  padding-bottom: 30px;
+.description-section__title {
+  padding: 20px 0 30px 0;
   font-size: 40px;
-  color: #252B42;
+  color: #252b42;
 }
-.interface-app__description{
-  width: 500;
+.description-section__description {
+  font-weight: 500;
   font-size: 24px;
   color: #737373;
   line-height: 38px;
-  padding-bottom: 154px;
-}
-.payment-info__description{
-  display: flex;
-  justify-content: space-evenly;
 }
-.payment-info__img{
-  display: flex;
-  justify-content: flex-start;
-  align-items: center;
-  height: 600px;
-  width: 302px;
+
+.section-benefits {
+  background-image: url('../images/Rectangle\ 47.png');
+  background-repeat: no-repeat;
+  background-size: cover;
+  max-height: 516px;
 }
-.payment-info__item{
+.section-benefit__text{
   display: flex;
   justify-content: center;
   flex-direction: column;
-  padding-bottom: 300px;
-  padding-left: 171px;
-}
-.payment-info__checkout{
-  padding-top: 150px;
-  padding-bottom: 25px;
-  width: 700;
-  color: #FA4A0C;
-  font-size: 24px;
-}
-.payment-info__advantages{
-  padding-top: 20px;
-  padding-bottom: 30px;
-  font-size: 40px;
-  color: #252B42;
-}
-.payment-info__deliver{
-  width: 500;
-  font-size: 24px;
-  color: #737373;
-  line-height: 38px;
-  padding-bottom: 154px;
-}
-.section{
-  background-image: url('../images/Rectangle\ 47.png');
-  background-repeat: no-repeat;
+  align-items: center;
   height: 516px;
-  background-size: cover;
+  width: 100%;
 }
-.section-top{
+.section-benefit__dwn {
   display: flex;
   justify-content: center;
-  padding-top: 150px;
   padding-bottom: 10px;
   font-weight: 700;
   font-size: 47px;
-  color: #FFFFFF;
+  color: #ffffff;
 }
-.section-row{
+.section-benefit__description {
   display: flex;
   justify-content: center;
   padding-top: 20px;
   padding-bottom: 30px;
   font-weight: 500;
   font-size: 24px;
-  color: #FFFFFF;
+  color: #ffffff;
 }
-.section-buttons{
+.section-buttons {
   display: flex;
   justify-content: center;
-  padding-bottom: 161px;
-  padding-top:40px;
+  padding-top: 40px;
 }
-.footer{
+.footer {
   display: flex;
-  justify-content: space-around;
+  justify-content: space-between;
 }
-.footer-logo{
+.footer-logo {
   display: flex;
   justify-content: flex-start;
-  align-items: center;
-  height: 246px;
-  padding-right: 250px;
+  align-items: flex-start;
+  flex-direction: row;
+  padding-bottom: 27px;
+  width: 206px;
 }
-.footer-icons{
+.footer-icons {
   display: flex;
-  justify-content: space-around;
+  justify-content: space-evenly;
   align-items: center;
-  padding-top: 100px;
-  height: 33px;
-  width: 33px;
+  height: 180px;
```

Comment:
Why height? I think it's not needed here. Please find another way to do right display.